config/config.go

```diff
@@ -91,10 +94,24 @@ func GetOptionalParameters(config *Config, defaultLidarDataRateMsec, defaultMapR
 		} else {
 			lidarDataFreqHz, err := strconv.Atoi(strCameraDataFreqHz)
 			if err != nil {
-				return 0, 0, newError("camera[data_frequency_hz] must only contain digits")
+				return 0, "", 0, 0, newError("camera[data_frequency_hz] must only contain digits")
 			}
 			lidarDataRateMsec = 1000 / lidarDataFreqHz
 		}
+		imuName, ok = config.MovementSensor["name"]
+		if ok {
+			strMovementSensorDataFreqHz, ok := config.MovementSensor["data_frequency_hz"]
+			if !ok {
+				logger.Debugf("config did not provide movement_sensor[data_frequency_hz], setting to default value of %d", 1000/defaultIMUDataRateMsec)
+			} else {
+				imuDataFreqHz, err := strconv.Atoi(strMovementSensorDataFreqHz)
+				if err != nil {
+					return 0, "", 0, 0, newError("movement_sensor[data_frequency_hz] must only contain digits")
+				}
+				imuDataRateMsec = 1000 / imuDataFreqHz
+			}
+		}
+		// should IMU data rate be set to 0 if no IMU is configured?
```

Comment:
Ah, I see what you mean now about whether to leave it as the default rate or set it to 0. I think it can be left as the default rate.